arm-gateway/2016-06-01/swagger/gateway.json

```diff
@@ -0,0 +1,442 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GatewayManagementClient",
```

Comment:
Feel free to use spaces here if you want, they'll get removed for code generation.